loglist/findlog/main.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// The findlog binary attempts to provide information about a log based on
+// ID or name.
+package main
+
+import (
+	"crypto/sha256"
+	"encoding/base64"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/google/certificate-transparency-go/loglist"
+)
+
+var (
+	logList = flag.String("log_list", loglist.LogListURL, "Location of master log list (URL or filename)")
```

Comment:
Possibly cleaner to have 2 flags log_file or log_url? I don't know. Or do url.Parse() and then file urls would work.

Reply:
I'd prefer to keep with a single argument because I'm trying to make the tool as simple to use as possible.

Reply:
(`file:` URLs don't work without manual splitting btw)

Reply:
Oh sorry, was thinking of Java :-)